src/Controller/Api/EntryRestController.php

```diff
@@ -528,7 +528,7 @@ public function postEntriesListAction(Request $request, EntryRepository $entryRe
         $limit = $this->getParameter('wallabag.api_limit_mass_actions');
 
         if (\count($urls) > $limit) {
-            throw new HttpException(400, 'API limit reached');
+            throw new BadRequestHttpException('API limit reached');
```

Comment:
changed to be more specific as needed to be allowed in config to preserve the message